lib/routes/baijing/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/article',
+    categories: ['new-media'],
+    example: '/baijing/article/',
+    url: 'www.baijing.cn/article/',
+    name: '白鲸出海 资讯',
```

Comment:
No need to repeat namespace name https://github.com/DIYgod/RSSHub/blob/63b98cfe90270be701edf6aa3ee1842a7ce54cc7/lib/routes/baijing/namespace.ts#L4 in route name